airflow/providers/ftp/hooks/ftp.py

```diff
@@ -58,7 +58,14 @@ def get_conn(self) -> ftplib.FTP:
         if self.conn is None:
             params = self.get_connection(self.ftp_conn_id)
             pasv = params.extra_dejson.get("passive", True)
-            self.conn = ftplib.FTP(params.host, params.login, params.password)  # nosec: B321
+            self.conn = ftplib.FTP()  # nosec: B321
+            if params.host:
+                port = ftplib.FTP_PORT
+                if params.port is not None:
+                    port = params.port
```

Comment:
Can we add a log statement here so that we know which port was used

Reply:
Hello, I added a log message. Can you take a look?